source/helpers.js

```diff
@@ -22,6 +24,27 @@ export const unzoom = id => {
   return id
 }
 
+const getSize = () => ({
+  innerHeight: window.innerHeight,
+  innerWidth: window.innerWidth,
+  outerHeight: window.outerHeight,
+  outerWidth: window.outerWidth
+})
+
+export const useWindowSize = () => {
+  const [windowSize, setWindowSize] = useState(getSize())
+  const handleResize = () => setWindowSize(getSize())
```

Comment:
Do you think this needs to be throttled?

Reply:
We weren't doing it before (https://github.com/rpearce/react-medium-image-zoom/blob/v3.x/src/EventsWrapper.js#L15)... I say we throttle when we need to.

Reply:
I actually just realized that this helpers file is going away (not using anything in it).

I'd move this to a different file.

Should we use https://github.com/streamich/react-use/blob/master/docs/useWindowSize.md, or do we need the different implementation?

Reply:
Good suggestion.  Nothing special.  Let's use the `react-use` one.